src/main/java/org/skife/jdbi/v2/BeanMapper.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2004 - 2014 Brian McCallister
+ * Copyright (C) 2004 - 2015 Brian McCallister and others
```

Comment:
Only reason I looked at license headers was because the Maven build barked at me if I left it out of any new files. Then muscle memory from my time as an Eclipse committer took over...

I'm not sure from your comment if you want me to change anything here.